Additional VRO plugin uninstall automation

erikPrime opened the pull request:
This commit adds a script that automates previously manual steps.

Commit:
Adds a script that automates previously manual steps.  Run delete_vro_plugin.rb before remove_plugin.sh

ghost commented:
This contains a known issue.  This script doesn't delete the package in Administer | Packages | com.puppet.011n.puppet.

Confusingly, it seems to manage to delete com.puppet.0-11n.puppet.node or something.

Commit:
This script actually deletes the package now.

The URL must end in a slash or it 404s.

Commit:
Added sshcmd, bash commands to complete uninstall

This commit adds sshcmd and bash commands needed to complete the uninstall.  Cleaned up some minor things.

Commit:
Polish

- Lined up some spaces.
- Cleared up the error message in delete_url.
- Removed pry.
- Set the default server back to TSE.

ghost commented:
The above known issue has been resolved.

ghost commented:
OK; Don't merge; Needs to have some kind of server name input, and a step where the script comes back and says "Are you sure you want to delete the Puppet VRO plugin from Server <serverName>"

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/puppetlabs/puppet-vro-starter_content?pullRequest=26) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/puppetlabs/puppet-vro-starter_content?pullRequest=26) before we can accept your contribution.<br/><hr/>**erikPrime** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/puppetlabs/puppet-vro-starter_content?pullRequest=26) it.</sub>
